src/app/pages/things/channels/details/channels.details.component.ts

```diff
@@ -109,12 +110,14 @@ export class ChannelsDetailsComponent implements OnInit {
   }
 
   getchannelMessages() {
-    this.messagesService.getMessages(this.channel.id, this.connections[0].key).subscribe(
-      (respMsg: any) => {
-        if (respMsg.messages) {
-          this.messages = respMsg.messages;
-        }
-      },
-    );
+    if (this.connections.length > 0) {
```

Comment:
You can simply put `if (this.connections.length)` since in js (and typescript) 0 is falsy.